src/core-runtime.js

```diff
@@ -30,6 +30,14 @@ Globalize.locale = function( locale ) {
 	return this._locale;
 };
 
+Globalize._loadRuntimeFn = function( key ) {
+	var fn = Globalize[ key ];
+	if ( typeof fn === "function" ) {
+		fn.runtimeKey = key;
+	}
+	return fn;
+};
```

Comment:
Thinking more about it...

Instead of stamping the functions in here, we could do that in the [compiler](https://github.com/globalizejs/globalize-compiler) itself (during build time).